.github/workflows/ci.yml

```diff
@@ -10,7 +10,6 @@ on:
 
 env:
   python-version: "3.9"
-  rust-version: 1.66.1
```

Comment:
This is unused and `rust-toolchain.toml` is used instead, right ?

Reply:
![image](https://user-images.githubusercontent.com/30697622/226322322-97c61ff0-5b63-429e-8074-3764f6df2a26.png)